maven-plugin-plugin/src/main/java/org/apache/maven/plugin/plugin/metadata/AddPluginArtifactMetadataMojo.java

```diff
@@ -42,8 +42,8 @@
  * <li>to define plugin mapping in the group</li>
  * </ol>
  *
- * @see ArtifactRepositoryMetadata
- * @see GroupRepositoryMetadata
+ * @see org.apache.maven.artifact.repository.metadata.ArtifactRepositoryMetadata
+ * @see org.apache.maven.artifact.repository.metadata.GroupRepositoryMetadata
```

Comment:
These links will not work if Javadoc does not find the deps on the classpath.

Reply:
Maven-compat is still in the dependencies afaik.  It has to be until we actually completely remove this mojo.